src/descriptors.h

```diff
@@ -74,19 +74,25 @@ struct local_orbital_descriptor
 {
     /// Orbital quantum number \f$ \ell \f$.
     int l;
-    
+
+    int j;
+
+    double total_angular_momentum;
     /// Set of radial solution descriptors.
-    /** Local orbital is constructed from at least two radial functions in order to make it zero at the 
+    /** Local orbital is constructed from at least two radial functions in order to make it zero at the
      *  muffin-tin sphere boundary. */
     radial_solution_descriptor_set rsd_set;
 };
 
 /// Descriptor of the pseudopotential.
 struct pseudopotential_descriptor
 {
+    /// The pseudo potential includes spin orbit coupling
+    bool SpinOrbit_Coupling{false};
```

Comment:
variables are in general in low letters:
SpinOrbit_Coupling -> spin_orbit_coupling